README.adoc

```diff
@@ -0,0 +1,100 @@
+image:https://godoc.org/github.com/codeready-toolchain/toolchain-e2e?status.png[GoDoc,link="https://godoc.org/github.com/codeready-toolchain/toolchain-e2e"]
+
+This repo contains e2e tests for https://github.com/codeready-toolchain/host-operator[host] and https://github.com/codeready-toolchain/member-operator[member] operators of CodeReady Toolchain.
+
+== End-to-End tests
+
+The e2e tests are executed against host and member operators running in OpenShift. The operators are built from the https://github.com/codeready-toolchain/host-operator[host-operator] and https://github.com/codeready-toolchain/member-operator[member-operator] repositories.
+
+Since the changes in e2e repo sometimes require also changes in some of the operator repositories, the logic that executes the e2e tests supports a feature of pairing PRs based on branch names.
+Before the e2e tests are executed in openshift-ci, the logic automatically tries to pair a PR opened for this (toolchain-e2e) repository with a branch of the same name that potentially could exist in any of the developer's fork of the operator repositories.
+
+For example, if a developer with GH account `cooljohn` opens a PR (for toolchain-e2e repo) from a branch `fix-reconcile`, then the logic checks if there is a branch `fix-reconcile` also in the `cooljohn/host-operator` and `cooljohn/member-operator` forks.
+Let say that `cooljohn/host-operator` contains such a branch but `cooljohn/member-operator` doesn't, then the logic:
+
+1. clones latest changes of both repos https://github.com/codeready-toolchain/host-operator[codeready-toolchain/host-operator] and https://github.com/codeready-toolchain/member-operator[codeready-toolchain/member-operator]
+2. fetches the `fix-reconcile` branch from `cooljohn/host-operator` fork
+3. merges `master` branch with the changes from `fix-reconcile` branch inside of `host-operator` repo
+4. builds images from the merge branch of `host-operator` repo and from `master` branch of `member-operator` repo & deploys them to OpenShift
+5. runs e2e tests taken from the opened PR
+
+It would work analogically also for the cases when the repositories either both or none of them contain the branch name.
+
+If you still don't know what to do with e2e tests in some use-cases, go to <<What to do>> section where all use-cases are covered.
+
+==== Prerequisites if running locally on Minishift
+If you are running these tests locally in Minishift, make sure that you have exposed Minishift's docker-env, so deployment can use the locally built image. You can expose it by running the following command.
+`eval $(minishift docker-env)`
+
+
+NOTE: This is not required for openshift-ci environment
+
+==== Running End-to-End Tests
+
+Although the e2e tests are in the separated repository that the actual operators are, it's still possible to run them against the current code that is at HEAD of the operator repositories.
```

Comment:
Although the e2e tests are in the separated repository ~~that~~ **than** the actual operators are

Reply:
fixed 463d37f

Reply:
https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request#applying-a-suggested-change just sayin'